pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -73,12 +80,19 @@ type pvAndPVCCountCollector struct {
 	pvLister PVLister
 	// Cache for accessing information about PersistentVolumeClaims.
 	pvcLister PVCLister
+	// Volume plugin manager
+	pluginMgr *volume.VolumePluginMgr
 }
 
 // Check if our collector implements necessary collector interface
 var _ metrics.StableCollector = &pvAndPVCCountCollector{}
 
 var (
+	totalPVCountDesc = metrics.NewDesc(
```

Comment:
What do you think of adding the phase and datasource kind as a metric label?

Reply:
Hm... I actually don't have any opinion on that. :) But if we agree it's useful to add them here, I don't mind doing it. These metrics are still alpha, I assume it's also for us to figure out which are useful and give us opportunity to make changes.

Reply:
I think they would be useful so you can also get a count of pvs in bound/available/released/lost phases, and datasource is useful so you can tell if it's a snapshot restore/clone/populator/etc